packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -91,8 +92,10 @@ export class Suggestions<T> extends BaseComponent<ISuggestionsProps<T>, {}> {
           { noResultsFoundText }
         </div> : null;
     };
-
-    let headerText = isMostRecentlyUsedVisible && mostRecentlyUsedHeaderText ? mostRecentlyUsedHeaderText : null;
+    // MostRecently Used text should supercede the header text if it's there and available.
+    let headerText = isMostRecentlyUsedVisible && mostRecentlyUsedHeaderText ?
+      mostRecentlyUsedHeaderText :
+      suggestionsHeaderText ? suggestionsHeaderText : null;
```

Comment:
can this be improved? it's hard for me to read this logic.

Reply:
Done.